pygmt/clib/session.py

```diff
@@ -930,39 +931,43 @@ def put_vector(self, dataset, column, vector):
                 f"in column {column} of dataset."
             )
 
-    def put_strings(self, dataset, family, strings):
+    def put_strings(
+        self,
+        dataset: ctp.c_void_p,
+        family: Literal["GMT_IS_VECTOR", "GMT_IS_MATRIX"],
```

Comment:
We can't use Literal here, since family can be `GMT_IS_VECTOR|GMT_IS_DUPLICATE` or similar
```suggestion
        family: str,
```